Flashcards.Deviationlark/Flashcards.cs

```diff
@@ -0,0 +1,88 @@
+using System.Configuration;
+using System.Data.SqlClient;
+using Dapper;
+
+namespace Flashcards
+{
+    class FlashcardsController
+    {
+        public string connectionString = ConfigurationManager.ConnectionStrings["connString"].ConnectionString;
+
+        internal List<FlashcardModel> ReadFlashcards(int id)
+        {
+            TableVisualisation tableVisualisation = new();
+            List<FlashcardModel> flashcards = new();
+            using (SqlConnection connection = new SqlConnection(connectionString))
+            {
+                connection.Open();
+
+                string query = $"SELECT * FROM Flashcards WHERE StackId = {id}";
+
+                flashcards = connection.Query<FlashcardModel>(query).ToList();
+
+                connection.Close();
```

Comment:
The using statement will take care of closing the connection. You can read more about it [here](https://learn.microsoft.com/en-us/dotnet/api/system.data.sqlclient.sqlconnection.close?view=dotnet-plat-ext-8.0#examples)